packages/mock-addon-docs/.storybook/main.js

```diff
@@ -1,25 +1,20 @@
-import remarkGfm from 'remark-gfm';
-
-export default {
+const config = {
     stories: [
-        '../stories/**/*stories.mdx',
-        '../stories/**/*stories.@(js|jsx|ts|tsx)',
+        '../stories/**/*.mdx',
```

Comment:
I believe this line can be removed as MDX files are no longer supported since Storybook v8.x

Reply:
Are you sure ? I can see that they removed the support for .stories.mdx https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#dropping-support-for-storiesmdx-csf-in-mdx-format-and-mdx1-support

Checking their docs for V8 they still use mdx and show examples of it 
https://storybook.js.org/docs/writing-docs/mdx